### ISSUE TITLE
language.html partial breaks build

### ISSUE THREAD

**Issue by martinbira** [squidfunk/mkdocs-material#516]
## Description

With 1.12.0 version the first line language.html
```
{% set languages = config.extra.language | default("en") %}
```
breaks mkdocs serve / build and returns the error

```
jinja2.exceptions.UndefinedError: 'config' is undefined
```

Copied language.hml into my theme partials folder and change line one to 
```
{% set languages = "en" %}
```
to verify behaviour. mkdocs serve / build worked as intended afterwards.

### Expected behavior

mkdocs serve / build runs

### Actual behavior

mkdocs serve / build does not run

"Full error":
```
 File "c:\python36\lib\site-packages\material\partials\language.html", line 1, in top-level template code
   {% set languages = config.extra.language | default("en") %}
 File "c:\python36\lib\site-packages\jinja2\environment.py", line 430, in getattr
   return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'config' is undefined
```

### Package versions

* Python: 3.6.1
* MkDocs: 0.16.3
* Material: 1.12.0

**Comment by squidfunk** [comment]
Have you overriden `base.html` or any other partials than `language.html`? The partial needs to be imported using `with context` or Jinja doesn't get `config` passed.

**Comment by martinbira** [comment]
toc.html had import without context. Fixed the problem

**Comment by squidfunk** [comment]
I'm sorry this release broke your build. It's the edge case where someone has overriden a template that includes `language.html`, but not `language.html?  itself. I will provide another check for downward compatibility in the next release

**Comment by squidfunk** [comment]
Fixed in #523

**Comment by squidfunk** [comment]
Released with 1.12.1
